### PR TITLE
Add matnwb

### PR DESCRIPTION
**Pull request by bendichter**
We are removing the webpage from the matnwb repo, so we want to add this to catch old links that still point to https://neurodatawithoutborders.github.io/matnwb

### CODE REVIEW

**Comment by rly** [comment]
This looks good to me, though it is a bit hard to tell what changes are part of this PR and not other PRs already merged to dev, because main needs to be merged with dev first.

**Review by rly — APPROVED** [review]

File: config/_default/menus.yaml [hunk 1]
```diff
@@ -51,6 +51,10 @@ main:
     weight: 6
     url: "working-groups/"
     parent: Community
+  - name: "MatNWB"
+    weight: 4
```
Comment (rly) [inline config/_default/menus.yaml:55]: I think `weight` is used for ordering and so it should be unique?
```suggestion
    weight: 7
```

**Comment by bendichter** [comment]
I actually don't know if we need the MatNWB page based on our meeting yesterday. Let's remove that for now and add it later if we need it. But @ehennestad this works, so if you want to design a page this is the way to do it

**Comment by bendichter** [comment]
working group changes moved to https://github.com/NeurodataWithoutBorders/neurodatawithoutborders.github.io/pull/9

**Pull request merged**